diff --git a/dist/client/app/composables/codeGroups.js b/dist/client/app/composables/codeGroups.js
index 94a2a86c36f4a32f19c73323b1198321e483c531..81294ebeee237761a7e0f1e89ec1146bdace6026 100644
--- a/dist/client/app/composables/codeGroups.js
+++ b/dist/client/app/composables/codeGroups.js
@@ -8,7 +8,7 @@ export function useCodeGroups() {
                 const group = el.parentElement?.parentElement;
                 const i = Array.from(group?.querySelectorAll('input') || []).indexOf(el);
                 const current = group?.querySelector('div[class*="language-"].active');
-                const next = group?.querySelectorAll('div[class*="language-"]')?.[i];
+                const next = Array.from(group?.querySelectorAll('div[class*="language-"]') || []).filter((val) => !val.className.match('language-id'))?.[i]
                 if (current && next && current !== next) {
                     current.classList.remove('active');
                     next.classList.add('active');